ledger/eval.go

```diff
@@ -54,6 +54,17 @@ const maxPaysetHint = 20000
 // transaction group.
 const asyncAccountLoadingThreadCount = 4
 
+type creatable struct {
+	cindex basics.CreatableIndex
+	ctype  basics.CreatableType
+}
+
+// FoundAddress is a wrapper for an address and a boolean.
+type FoundAddress struct {
```

Comment:
why is it exported? The struct name sound a bit weird. Maybe `cachedCreator` or something?

Reply:
`FoundAddress` is also used in `evalIndexer.go` where this type needs to be exposed to Indexer. I think this name is ok for caching as well. `FoundAddress` encodes an address that is found (or not found). That is exactly what we want to cache.